onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -33,39 +33,29 @@ BackendManager::BackendManager(const onnxruntime::Node* fused_node, const loggin
 
   // Save the indexes of graph inputs among fused_node's inputDefs
   // (which also contains initializers).
-  #if (defined OPENVINO_2020_2) || (defined OPENVINO_2020_3)
-    std::map<std::string, int> inputdef_index_map;
-  #endif
   auto node_input_defs = fused_node->InputDefs();
   int i = 0;
   for (auto idef : node_input_defs) {
-    #if (defined OPENVINO_2020_2) || (defined OPENVINO_2020_3)
-    inputdef_index_map.insert({idef->Name(), i});
-    #else
     subgraph_context_.input_names.insert({idef->Name(), i});
-    #endif
     i++;
   }
 
   auto graph_inputs = fused_node->GetFunctionBody()->Body().GetInputs();
   for (auto input : graph_inputs) {
-    if(GetGlobalContext().device_type == "MYRIAD"){
+    if(GetGlobalContext().device_type.find("MYRIAD") != std::string::npos){
```

Comment:
>GetGlobalContext().device_type.find("MYRIAD") [](start = 7, length = 45)

should we consider defining enum or bitwise type instead of string to make it faster? same for precision_str. string comparison/search is slow.

Reply:
@HectorSVC I think it's fine as it's a one time job. Also now that with Hetero there can be many number of device_type combinations possible, I think it would be appropriate to stick with the strings.